task/__init__.py

```diff
@@ -321,7 +321,8 @@ def censored(text):
         if verbose:
             sys.stderr.write("! " + str(e))
         raise
-    sys.stderr.write(censored(output))
+    if output:
```

Comment:
How can this not  exist? the `except` handler re-raises the exception, so it should never get here?

Reply:
This is true.  And indeed, it didn't fix my problem that we're getting 

```
  File "/home/runner/work/cockpit/cockpit/src/bots/task/__init__.py", line 325, in execute
    sys.stderr.write(censored(output))
  File "/home/runner/work/cockpit/cockpit/src/bots/task/__init__.py", line 312, in censored
    return text.replace(api.token, "CENSORED")
TypeError: replace() argument 1 must be str, not None
```

so it must be something else.

Reply:
Oh.  It's the `api.token` that's NULL.  Duh.